backend/spotihue/oauth.py

```diff
@@ -1,23 +1,33 @@
 import logging
 import webbrowser
-from urllib import parse
 from typing import Optional
+from urllib import parse
 
 from spotipy import oauth2, util
 
 
 logger = logging.getLogger(__name__)
 
+SPOTIFY_OAUTH_TIMEOUT = 60 * 3  # 3 minutes
```

Comment:
Should we create a .env for this to make it configurable?

Reply:
good call! will do